c2corg_ui/static/js/documentediting.js

```diff
@@ -551,12 +586,28 @@ app.DocumentEditingController.prototype.formatOuting_ = function(outing) {
     // init empty conditions_levels for ng-repeat
     outing['locales'][0]['conditions_levels'] = [{'level_snow_height_soft': '', 'level_snow_height_total': '', 'level_comment': '', 'level_place': ''}];
   }
-
   return outing;
 }
 
 
 /**
+ * get parameters from the URL separated by &
+ * @returns {Object} url parameters
+ * @private
+ */
+app.DocumentEditingController.prototype.parseUrl_ = function() {
```

Comment:
OK but I think it's not the right way to do it. In ``ngeo`` there is a ``ngeo.Location`` service that can be used to get/update/delete the URL parameters.
Of course here we are not in a map context, but the service is still available and will be used in other pages (in the advanced search pages I guess).
See https://camptocamp.github.io/ngeo/master/apidoc/ngeo.Location.html and http://camptocamp.github.io/ngeo/master/examples/permalink.html